cilium/cmd/bpf_policy_get.go

```diff
@@ -97,14 +97,12 @@ func listMap(args []string) {
 }
 
 func dumpMap(file string) {
-
-	fd, err := bpf.ObjGet(file)
+	m, err := policymap.Open(file)
 	if err != nil {
-		Fatalf("%s\n", err)
+		Fatalf("Failed to open map: %s", err)
```

Comment:
Shouldn't there still be a newline character here?

Reply:
Good catch.